nixos/modules/services/databases/riak-cs.nix

```diff
@@ -110,13 +118,6 @@ in
         '';
       };
 
-      extraAdvancedConfig = mkOption {
```

Comment:
Is this option no longer required? Do you already cover all options in `riak-cs/advanced.config`?